packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -470,10 +471,47 @@ export default class Meetings extends WebexPlugin {
      * @param {String} [options.feedbackId] ID used for tracking
      * @param {String} [options.locusId]
      * @param {Stirng} [options.correlationId]
+     * @param {String} [options.meetingId]
      * @returns {String} feedback ID logs were submitted under
      */
     uploadLogs(options = {}) {
-      return this.loggerRequest.uploadLogs(options);
+      if (this.config.autoUploadLogs) {
+        LoggerProxy.logger.info('Meetings:index#uploadLogs --> Meeting destroyed, uploading logs');
```

Comment:
This log is incorrect, we don't know the cause of why we are uploading logs.

```suggestion
        LoggerProxy.logger.info('Meetings:index#uploadLogs --> Uploading logs');
```